src/uu/stdbuf/src/stdbuf.rs

```diff
@@ -102,7 +103,10 @@ fn check_option(matches: &ArgMatches, name: &str) -> Result<BufferType, ProgramO
                 }
             }
             x => parse_size_u64(x).map_or_else(
-                |e| crash!(125, "invalid mode {}", e),
+                |e| {
+                    set_exit_code(125);
```

Comment:
I think this lines should not be necessary. The exit code should be added to the error here:

https://github.com/uutils/coreutils/blob/26df9445fb70152abfd523c0bdb75def6c7d5475/src/uu/stdbuf/src/stdbuf.rs#L151

It'd be great if you could also verify that this is actually the case by writing a test for it.

Reply:
Ah, you're right, I've taken that line out. However, the test already asserts that the error code is 125: https://github.com/cswn/coreutils/blob/26df9445fb70152abfd523c0bdb75def6c7d5475/tests/by-util/test_stdbuf.rs#L67C9-L67C9